lib/resources/r4/encounter.yaml

```diff
@@ -527,6 +527,31 @@ fields:
         "reference": "Organization/619848"
       }
     }
+  note: >
+    <ul>
+      <li>serviceProvider cannot be provided when Encounter.location.location is present.</li>
```

Comment:
nit: let's put field names in ```code```.

Reply:
Also we should be consistent with our naming. Either both should be 'serviceProvider' and 'location' or both should be 'Encounter.serviceProvider' and 'Encounter.location'.

Reply:
[done](https://github.com/cerner/fhir.cerner.com/pull/803/commits/62e556c0c21d59fccf0d1ab62663a549c0045d74#diff-ea755c3df7ed8869a54a274cedf2e40e119b2f16a6e3ed8bc8ae211e0e87389cR532)